src/pages/api/users/forgotpassword.js

```diff
@@ -0,0 +1,17 @@
+import { forgotPassword } from "../../../../server/mongodb/actions/ForgotPassword";
+
+const handler = (req, res) => {
+  if (req.method === "POST") {
```

Comment:
You should return [status code 405](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/405) for non-POST calls.